pilot/pkg/serviceregistry/serviceentry/servicediscovery_test.go

```diff
@@ -1313,10 +1313,9 @@ func Test_autoAllocateIP_conditions(t *testing.T) {
 			},
 			wantServices: []*model.Service{
 				{
-					Hostname:             "foo.com",
-					Resolution:           model.DNSLB,
-					Address:              "0.0.0.0",
-					AutoAllocatedAddress: "240.240.0.1",
```

Comment:
I'm curious: is there a chance of collision with real IP addresses here and breaking security policies over IPs (network or authz)?